src/app/event/Modal.tsx

```diff
@@ -40,14 +42,25 @@ export function Modal() {
     name: "addresses",
   });
 
+  const createTicketMultiAttestations = useCreateTicketMultiAttestations();
+
   const watchedFields = watch("addresses");
 
   function submitForm(data: FormData) {
-    const filteredData = data.addresses.filter(
+    const nonEmptyAddresses = data.addresses.filter(
       (element) => element.address !== ""
     );
-    console.log("Form submitted with data:", filteredData);
+    console.log("Form submitted with data:", nonEmptyAddresses);
     setOpen(false);
+
+    const tickets = nonEmptyAddresses.map((element) => {
+      return {
+        owner: element.address as Address,
+        eventId: eventId,
+      };
+    });
+
+    createTicketMultiAttestations({ tickets: tickets });
```

Comment:
There's some error chances here? If so, you should handle it here and maybe only closes the modal after success. (if so, consider add a loading state on the modal submit button too)

Reply:
The button triggers a call to another screen, it has a pretty small chance to have some error and the project has bigger priorities, so this will be on backlog